traffic_ops/app/lib/MojoPlugins/Health.pm

```diff
@@ -159,6 +159,9 @@ sub register {
 				cachegroups  => [],
 			};
 
+			$self->app->log->debug("get_cache_health() - " . 
```

Comment:
I think that a good practice would be to leave debug messages in the code, and set the default log level, at least in production, to "info", so that debug messages will not clutter production logs.

Since the current log level (defined in log4perl.conf) is "debug" (in all environments) I'd suggest to to change this message's log level from "debug" to "trace".